src/main/java/com/ait/lienzo/client/core/shape/wires/handlers/impl/WiresConnectorControlImpl.java

```diff
@@ -413,135 +438,62 @@ public boolean adjust(final Point2D dxy) {
         }
 
         @Override
-        public void onNodeDragEnd(final NodeDragEndEvent event) {
+        public void onNodeDragEnd(final NodeDragEndEvent event)
+        {
             final boolean allowed = connectionControl.onMoveComplete();
 
             // Cancel the drag operation if the connection operation is not allowed.
-            if (!allowed) {
+            if (!allowed)
+            {
                 event.getDragContext().reset();
             }
             //decorate connection shape
             final ShapeState shapeState = (connection.isAutoConnection() || connection.getMagnet() != null ?
-                    ShapeState.VALID : ShapeState.INVALID);
+                                           ShapeState.VALID : ShapeState.INVALID);
             m_pointHandleDecorator.decorate(shape, shapeState);
         }
     }
 
-    private void refreshControlPoints() {
+    private void refreshControlPoints()
+    {
         hideControlPoints();
         showPointHandles();
     }
 
-    private void batch() {
-        if (null != m_wiresManager.getLayer().getLayer()) {
+    private void batch()
+    {
+        if (null != m_wiresManager.getLayer().getLayer())
+        {
             m_wiresManager.getLayer().getLayer().batch();
         }
     }
 
-    private IControlPointsAcceptor getControlPointsAcceptor() {
+    private IControlPointsAcceptor getControlPointsAcceptor()
+    {
         return m_wiresManager.getControlPointsAcceptor();
     }
 
-    private static boolean isConnected(final WiresConnection connection) {
+    private static boolean isConnected(final WiresConnection connection)
+    {
         return null != connection && null != connection.getMagnet();
     }
 
     @Override
-    public void destroy() {
+    public void destroy()
+    {
         clear();
         destroyPointHandles();
     }
 
-    public void setPointHandleDecorator(PointHandleDecorator pointHandleDecorator) {
+    public void setPointHandleDecorator(PointHandleDecorator pointHandleDecorator)
+    {
         this.m_pointHandleDecorator = pointHandleDecorator;
     }
 
-    /**
-     * Timer class responsible to delay simulating a long press when click + drag to add a new
-     * control point. Besides that the delay is necessary to preserve the selection on connector
-     * when there is only a click, in this case this time should be cancelled.
-     */
-    protected class AddControlPointTimer extends Timer {
```

Comment:
I think this timer was making the connector selection possible without creating a new bend point.

Reply:
Yeahh fixed, thanks! :+1: